### DISCUSSION TITLE
How to move from Google Santa to this one?

### DISCUSSION THREAD

**Discussion by e40** [northpolesec/santa#115] [Q&A] [answered]
I downloaded the latest (2024.10), but after installing I still see this:

```
@max[git:master]$ santactl version
santad          | 2024.9 (build 674285143)
santactl        | 2024.9 (build 674285143)
SantaGUI        | 2024.9 (build 674285143)
@max[git:master]$ 
```

Do I need to uninstall the old one before installing the new one?  Reboot?

**Comment by e40** [comment]
Also, the date on `/Applications/Santa.app/` is when I installed 2024.9.

**Comment by russellhancox** [comment, marked as answer]
We have documentation on how to do this here: https://northpole.dev/deployment/migration

**Reply by e40** [reply]
I don't use an MDM, but dragging Google Santa to the trash gives me "The item “Santa.app” can’t be moved to the Trash because it’s open."

**Reply by russellhancox** [reply]
You can quit the existing Santa.app by running `launchctl remove com.google.santa`, then drag to the trash.

**Reply by e40** [reply]
I looked at the "Getting Started" docs and I don't remember ever doing this stuff with the original Google Santa (I realize the new docs are just a clone of the old docs right now).

```
@max[git:master]$ santactl version
santad          | 2024.10 (build 59, commit 3aefacbd)
santactl        | 2024.10 (build 59, commit 3aefacbd)
SantaGUI        | 2024.10 (build 59, commit 3aefacbd)
@max[git:master]$ sudo santactl rule --export rules.json
An error occurred communicating with the daemon, is it running?
@max[git:master]$
```

Ideas on that?  Presumably I need a configuration profile?  As I said, never had one before.

My use case is to prevent certain programs from running.  Mail.app, Stocks.app and ironically the Google Chrome updater.

Ideas on how to proceed?

**Reply by e40** [reply]
The problem was "Full Disk Access" was not turned on for `com.northpolesec.santa.daemon`.

**Reply by e40** [reply]
The `export` worked and it has all my old rules.  👍

**Reply by e40** [reply]
OK, it has the old rules, but they don't work.  I guess I need to clean them out and make them again.

**Reply by pmarkowsky** [reply]
@e40 you should be able to import and export rules as JSON when you're not using a sync service. 

If that's not working please file an issue with us.
